src/views/routes/users.js

```diff
@@ -10,13 +12,16 @@ const {
   AUTH__EMAIL_DOES_NOT_EXIST,
   AUTH__PASSWORD_DOES_NOT_MATCH,
   AUTH__EMAIL_ALREADY_IN_USE,
+  PASSWORD_RESET_LINK_INVALID_OR_EXPIRED,
 } = constants.errorCodes;
 
 const {
   error: {
-    unauthorized,
+    badRequest,
     conflict,
+    notFound,
     serverError,
+    unauthorized,
```

Comment:
👍 alphabetized

Reply:
Haha, learned from many of your previous comments ;)